drivers/pcie/host/msi.c

```diff
@@ -6,6 +6,7 @@
 
 #include <kernel.h>
 #include <drivers/pcie/msi.h>
+#include <drivers/pcie/cap.h>
```

Comment:
Could you put these 3 last commits into their own PR? They are not specifically related to PCIe host controller support, and they could get merged faster as well.